src/FunctionApps/python/IntakePipeline/README.md

```diff
@@ -0,0 +1,9 @@
+# Intake Pipeline Function App
+
+## Config
+
+This function app requires some config to be present. In production this is defined on the function app, and locally, you'll want to add
+items in `local.settings.json` under `Values`.
+
+* `INTAKE_CONTAINER_URL`: a container URL containing ndjson formatted FHIR records (eg: https://pitestdatasa.blob.core.windows.net/bronze)
+* `INTAKE_CONTAINER_PREFIX`: a string prefix so we're not scanning the entire container (eg: decrypted/valid-messages)
```

Comment:
Are there circumstances where we'd want to process only a subset of the files in a container through the pipeline? I'm imagining we'll be pretty meticulous about what files we put in the decrypted containers, so why not just scan the whole container and be careful with file maintenance outside the pipeline?

Reply:
that bucket has all the encrypted files as well, so this gives us a little flexibility - also, for example, if we wanted to re-run just ELRs